pkg/image/validation/kubernetes_test.go

```diff
@@ -140,33 +140,38 @@ func TestIsKubernetesDefined(t *testing.T) {
 	assert.False(t, result)
 }
 
-func TestValidateNodes(t *testing.T) {
+func TestValidateNetwork(t *testing.T) {
+	var validV4Network = validNetwork
```

Comment:
Nit: Why do we copy the value instead of just using it?

Reply:
it's actually a leftover, as I initially had v4 & v6 variables. I have removed it, I'd change `validNetwork ` to `validV4Network `though.